Decrease infra deployment time drastically, update reamde (#1)
Azure-Samples/HttpRouteConfigBicep: ReadMe.md

- `prefixRewrite` the URL path to modify the URL prefix path to
- `targets` takes one target application for which the routes in a rule will be forwaded to. These can only be container app names.

### Optional: Provisioning a Managed TLS/SSL Certificate for HTTPS

> If you are using a **custom domain** in your `httpRouteConfig` and wish to access it over **HTTPS**, you need to provision a **managed TLS/SSL certificate** in Azure Container Apps. If you have binded your custom domain to the path based routing spec using `auto` bindingType then once the TLS certificate is provisioned successfully, we will automatically update the certificate id field in the spec and bind it to your routing config within the managed environment.

Reviewer: ```suggestion
If you are using a **custom domain** in your `httpRouteConfig` and wish to access it over **HTTPS**, you need to provision a **managed TLS/SSL certificate** in Azure Container Apps. If you have bound your custom domain to the routing spec using `auto` bindingType then once the TLS certificate is provisioned successfully, it will automatically be added to the certificate id field in the spec and be bound to your routing config within the managed environment.
```

 
#### Steps:
 
1. **Ensure DNS is properly configured**  

>    Your custom domain must have a valid DNS record pointing to your Azure Container App's ingress endpoint (typically a TXT record and a IP address of the Managed Environment).

Reviewer: + Doc link

> 2. **Add the `managedCertificate` resource to your Bicep template**  

Reviewer: Which file and order in this sample should this be added? In main.bicep after httpRoutes?
